.github/workflows/tests.yml

```diff
@@ -0,0 +1,404 @@
+name: GitHub Actions Demo
+run-name: OPEN RUNTIMES TEST
+on: [push]
```

Comment:
We need to run on pull_requests on tag creation. please check the right flags for it

Reply:
On push should cover pull requests. Before making pull request, you must push into a branch.

Yes, I agree we need something for when tag is released (to deploy)

Reply:
@Meldiron @christyjacob4 I have mentioned to requested changes below. What do you guys think of it?